iotqatools/cb_v2_utils.py

```diff
@@ -430,7 +430,7 @@ def __create_metadata(self, metadata_number, metadata_name, metadata_type, metad
                 meta_dict[name] = {}
                 if metadata_value is not None:
                     meta_dict[name][VALUE] = metadata_value
-                if metadata_type is not None:
+                if metadata_type != NONE:
```

Comment:
this `NONE` is a constant (`NONE = u'none`), and it is used in `type`, `attribute_type` or `metadata_type` as default value. 

`NONE` is not the same that None (null in python)